### ISSUE TITLE
"brave init" fails on new installation (darwin/arm64)

### ISSUE THREAD

**Issue by herrbuerger** [bravetools/bravetools#214]
**Describe the bug**
I've downloaded the latest build from Github and ran it for the first time. It resulted in the following error:

```
➜  ~ brave init
2022/11/28 11:50:21 Initialising a new Bravetools configuration
Host OS:  darwin
Backend:  multipass
Storage (GB):  12
Memory:  4GB
Network:  10.0.0.1
2022/11/28 11:50:21 Initialising Bravetools backend
launch failed: instance "christian" already exists
```

**To Reproduce**
Unclear how to reproduce. I didn't have a Bravefile yet.

**Expected behavior**
I expected brave to finish the init phase successfully.

**Environment (please complete the following information):**
 - Your operating system name and version: Ventura, 13.0.1 (22A400)
 - Bravetools version: 1.56
 - LXD version: don't know

**Comment by idroz** [comment]
Thanks. I believe this relates to your machine already having a VM named $USER. 

@Szubie : We need to make sure that the multipass is as sandboxed as possible...e.g. name = bravetools?

@herrbuerger Meanwhile, you could try installing bravetools using a configuration file. For example.

1. Create `config.yaml`
2. Populate the config.yaml with:

```bash
name: brave
trust: brave
profile: brave
storage:
  type: zfs
  name: brave-20221014072021
  size: 98GB
network:
  name: bravebr0
  ip: 10.0.0.1
backendsettings:
  type: multipass
  resources:
    name: brave
    os: bionic
    cpu: "8"
    ram: 12GB
    hd: 100GB
    ip: 192.168.64.65
status: active
remote: local 
```

3. run: `brave init --config config.yaml`

**Comment by herrbuerger** [comment]
Thanks, I gave that a try and ran into the following error while running the `init`:

```
Error: Failed to parse the preseed: yaml: unmarshal errors:
  line 1: field pools not found in type api.InitPreseed
2022/11/29 13:48:14 error initializing Bravetools backend: Failed to initiate workspace: exit status 1
```

**Comment by idroz** [comment]
Thanks - are you using the latest master branch or bravetools binary from releases?

**Comment by idroz** [comment]
no, no worries. I just managed to reproduce the error. Must have crept in in the latest LXD version bravetools is trying to install. Will look into it.

**Comment by idroz** [comment]
Auto closed by #215 but feel free to reopen if it doesn't fix your issue.

Could you try building locally from source and ping back if works ok?

**Comment by idroz** [comment]
Closing as addressed in #215 #216
